api/capi/include/nnstreamer.h

```diff
@@ -238,6 +244,19 @@ typedef void (*ml_pipeline_sink_cb) (const ml_tensors_data_h data, const ml_tens
  */
 typedef void (*ml_pipeline_state_cb) (ml_pipeline_state_e state, void *user_data);
 
+/**
+ * @brief Callback for custom condition of tensor_if
+ * @since_tizen 6.5
+ * @param[out] info The handle of tensors information (cardinality, dimension, and type of given tensor/tensors).
+ * @param[out] data The handle of the tensor output (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
```

Comment:
output? not an input?
out --> in ?

Reply:
@myungjoo  Nice Catch! We need to update it.

Reply:
Thanks! Updated.
I will upload other related revisions.